openproblems/tasks/_batch_integration/batch_integration_embed/metrics/pcr.py

```diff
@@ -0,0 +1,13 @@
+from .....tools.decorators import metric
+
+
+@metric(
```

Comment:
missing method description, which was pasted into the CC conservation metric above.